src/mappings/entities/mapAsset.ts

```diff
@@ -244,20 +246,21 @@ const handleIssued = async (
 };
 
 const handleRedeemed = async (blockId: string, params: Codec[]): Promise<void> => {
-  const [, rawTicker, , rawAmount] = params;
+  const [, rawTicker, rawBeneficiaryDid, rawAmount] = params;
```

Comment:
```suggestion
  const [, rawTicker, rawIssuerDid, rawAmount] = params;
```
(nit) looks more consistent after getting parsed.

Reply:
Few other places like this. OK to leave if thats whats called on the event though

Reply:
The event said it to be beneficiary, so I left it like that